### ISSUE TITLE
ValueError: unknown url type: ''

### ISSUE THREAD

**Issue by saintech** [magnusjjj/gmadget#2]
```
C:\Users\home\Desktop\gmadget-master>gmadget --singleids 731156538

C:\Users\home\Desktop\gmadget-master>echo off
Active code page: 65001
Downloading
Traceback (most recent call last):
  File "gmaddown_console.py", line 121, in <module>
    urllib.request.urlretrieve(item["file_url"], file_savename)
  File "urllib\request.py", line 187, in urlretrieve
  File "urllib\request.py", line 162, in urlopen
  File "urllib\request.py", line 450, in open
  File "urllib\request.py", line 268, in __init__
  File "urllib\request.py", line 294, in full_url
  File "urllib\request.py", line 323, in _parse
ValueError: unknown url type: ''
```

**Comment by magnusjjj** [comment]
Hi! Thanks for the bug report. Taking a look at it :).

**Comment by magnusjjj** [comment]
I took a look at it, and could replicate your problem. I took a look at what the Steam API reports back, and for that particular steam collection it does not send back a file url to download. It would seem that StarBound does something magical on its own.

Garrys Mod files still work, so.. uh. Adding a clarification on the front page that its the only type guaranteed to work :). Sorry, but digging through the steam api shows that there is no other way to grab it.

There WAS this thing I found while googling, if you are trying to set up a server on linux:

https://www.reddit.com/r/starbound/comments/4u69z8/10_installing_steam_workshop_mods_on_servers/
